src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -260,7 +261,7 @@ void CheckELUDerivativeCorrect(const arma::colvec input,
 
 /*
  * Implementation of the PReLU activation function test. The function
- * is implemented as PReLU layer in the file perametric_relu.hpp
+ * is implemented as PReLU layer in the file parametric_relu.hpp
```

Comment:
I was going through the other tests in this file and found this small typo and fixed it. Hope it helped.

Reply:
ohh nice catch. Thanks.